pkg/util/crypto.go

```diff
@@ -53,9 +53,10 @@ func GenerateSelfSignedCert(host, certPath, keyPath string, alternateIPs []net.I
 		NotBefore: time.Now(),
 		NotAfter:  time.Now().Add(time.Hour * 24 * 365),
 
-		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature,
+		KeyUsage:              x509.KeyUsageKeyEncipherment | x509.KeyUsageDigitalSignature | x509.KeyUsageCertSign,
```

Comment:
I'm not sure we want to make all self signed certs into CA certs that have signing powers. That seems a bit overly broad. 

Since this is running a local cluster, can you not just generate certs locally (outside of the apiserver) and put them on the right spot in the filesystem? This is essentially what we do for GCE except that we push them onto the remote nodes via the metadata server. It'd be way easier if we just moved them to a local directory or passed a flag to the apiserver instead.